lips/lip-2.md

```diff
@@ -0,0 +1,126 @@
+|RotateAuthenticationKey(addr)                                                                                                                       |Account at addr             |Per address                                                                                            |Y             |
+|WithdrawalCapability(addr)                                                                                                                          |Account at addr             |Per address                                                                                            |Y             |
+
+
+* MintCurrency(**type**): Create currency of the given **type**
+* BurnCurrency(**type**): Destroy currency of the given **type**
+* PreburnCurrency(**type**): Create a burn request for the currency of the given **type** to be fulfilled or cancelled by the holder of BurnCurrency
+* UpdateExchangeRate(**type**): Update the exchange rate from each non-LBR currency **type** to LBR
+* UpdateDualAttestationLimit: Update the amount above which VASP <-> VASP transactions must use the travel rule protocol
+* {Freeze,Unfreeze}Account: Prevent an account from sending transactions or receiving funds
+* RegisterNewCurrency: Add a new currency type to the system
+* SendWriteSetTransaction: Used to update the Libra protocol by changing on-chain Move modules and/or global state
+* UpdateLibraProtocolVersion: Change the current Libra protocol version. This will atomically update the behavior of node software
+* UpdateVMConfig: Update the transaction script allowlist, module publishing options, or gas costs
+* PublishModule: Publish a new Move module
+* {Add,Remove}Validator: Add and remove validators
+* UpdateValidatorConfig(**addr**): Rotate the consensus public key and network address for the validator account at **addr**
+* SetValidatorOperator(**addr**): Set the address of the operator account that has permission to rotate the consensus public key of the validator account at **addr**
+* RotateDualAttestationInfo(**addr**): Change the public key and endpoint URL that addr uses for the KYC and travel rule protocols
+* RotateAuthenticationKey(**addr**): Rotate the authentication key for the account at **addr**
+* WithdrawCapability(**addr**): Decrease the balance of the account at **addr**
+
+All privileges are granted to an account upon creation, and most remain in that account forever. There are three exceptions: UpdateValidatorConfig, RotateAuthenticationKey and Withdraw.
+* UpdateValidatorConfig is granted to a Validator account upon creation, but can be delegated to a ValidatorOperator. The Validator can subsequently revoke the privilege and delegate to a different operator.
+* Both RotateAuthenticationKey and Withdraw can be be extracted from their original account and placed in a resource elsewhere (including one published under a different account).
+
+### Move Implementation
```

Comment:
It would be great if we could somehow connect each element of this policy document to a corresponding formal specification eventually. By the way, arguing that "permission check is implemented correctly" is in-scope of this document?

Reply:
> It would be great if we could somehow connect each element of this policy document to a corresponding formal specification eventually

Agreed! I think "special cases" requiring side explanation like `UpdateValidatorConfig` and `UpdateValidatorOperator` are indicators of a design that's not as simple as it could be. We could enrich the spec language, but we could (and perhaps should) also simplify the design.